s.to/dist/presence.js

```diff
@@ -0,0 +1,298 @@
+var presence = new Presence({
+    clientId: "463000750193246209",
+}), strings = presence.getStrings({
```

Comment:
remove comma after client id line, and fix deepscan errors

Reply:
Update your fork @MauriceX24 to fix the deepscan errors.